app/assets/javascripts/modules/coronavirus-local-restriction-results.js

```diff
@@ -0,0 +1,23 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function CoronavirusLocalRestrictionResults () {}
```

Comment:
Since this is only handling event tracking, I would suggest we name it something like `CoronavirusTrackLocalRestrictionResults` for future us to remember that there are no UI enhancement on results and we're only setting up tracking.

Reply:
Done